ReactNativeTraining/src/app/screens/Home/screens/Chats/components/Message/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
```

Comment:
This component would be in `src/app/screens/components/..` `Message` component is going to need for Chats, Groups and Contancts chats